app/views/translation/project.js

```diff
@@ -1,7 +1,7 @@
 ﻿define(['app', 'lodash'], function (app, _) {
     return ['$scope', '$http', '$q', '$routeParams',
     function ($scope, $http, $q, $routeParams) {
-            var baseUrl = window.baseUrl||'/';
+            var baseUrl = $scope.baseUrl = window.baseUrl||'/';
```

Comment:
trailing / is mandatory in this case.
may be you should ensure it's there

```suggestion
            var baseUrl = $scope.baseUrl = window.baseUrl.rerplace(/\/*$/, '') + '/';
```

Reply:
while this is somehting that comes from out ENV variables, i will move it to .ejs to i dont have to do it in all files,, ` window.baseUrl = '<%=baseUrl.replace(/\/*$/, "") + "/" %>';`